### ISSUE TITLE
missing 'box' causes panic

### ISSUE THREAD

**Issue by gregzuro** [wercker/wercker#164]
Running a pipeline that has no 'box' in force causes a panic at:

https://github.com/wercker/wercker/blob/master/docker/pipeline.go#L47

```
# (no 'box' above here)

# Run this to create the rc and service
kube-initialize:
    steps:

        - kubectl:
            server: $KUBERNETES_MASTER
            username: $KUBERNETES_USERNAME
            password: $KUBERNETES_PASSWORD
            insecure-skip-tls-verify: true
            command: create -f cities-controller.json

        - kubectl:
            server: $KUBERNETES_MASTER
            username: $KUBERNETES_USERNAME
            password: $KUBERNETES_PASSWORD
            insecure-skip-tls-verify: true
            command: create -f cities-service.json
```

```
16:09 $ s wercker check-config --pipeline kube-initialize
--> No Docker host specified, checking: /var/run/docker.sock
panic: runtime error: invalid memory address or nil pointer dereference
[signal 0xb code=0x1 addr=0x0 pc=0x61aec7]

goroutine 1 [running]:
panic(0xaeeb00, 0xc82000e100)
	/usr/local/go/src/runtime/panic.go:481 +0x3e6
github.com/wercker/wercker/docker.NewDockerPipeline(0xc82000e478, 0x5, 0xc820014140, 0xc820142600, 0xc8201960f0, 0x7f2908bc9268, 0xfccfe0, 0x0, 0x0, 0x0)
	/go/src/github.com/wercker/wercker/docker/pipeline.go:47 +0x217
github.com/wercker/wercker/cmd.cmdCheckConfig(0xc820142600, 0xc8201960f0, 0x0, 0x0)
	/go/src/github.com/wercker/wercker/cmd/main.go:452 +0x4f2
github.com/wercker/wercker/cmd.glob.func3(0xc8200e46c0)
	/go/src/github.com/wercker/wercker/cmd/main.go:124 +0x7b4
github.com/wercker/wercker/vendor/github.com/codegangsta/cli.Command.Run(0xbf0560, 0xc, 0x0, 0x0, 0xc72640, 0x18, 0x0, 0x0, 0x0, 0x0, ...)
	/go/src/github.com/wercker/wercker/vendor/github.com/codegangsta/cli/command.go:113 +0x10b0
github.com/wercker/wercker/vendor/github.com/codegangsta/cli.(*App).Run(0xc820100ea0, 0xc8200700c0, 0x4, 0x4, 0x0, 0x0)
	/go/src/github.com/wercker/wercker/vendor/github.com/codegangsta/cli/app.go:156 +0xbe9
main.main()
	/go/src/github.com/wercker/wercker/main.go:25 +0x49
```

**Comment by gregzuro** [comment]
NB: in this case, the GUI fails silently.  Maybe a different issue for that?

**Comment by Tonnu** [comment]
This was resolved by #184 I believe
